### PR TITLE
Fix PR #150 when key_url starts with "salt://"

### PR DESCRIPTION
**Pull request by bbinet**
See: https://github.com/salt-formulas/salt-formula-linux/pull/150#issuecomment-395805799

### CODE REVIEW

**Comment by bbinet** [comment]
I've rebased this PR on latest master to fix the conflict.

File: linux/system/repo.sls [hunk 1]
```diff
@@ -132,6 +132,9 @@ linux_repo_{{ name }}:
   {%- if repo.key_server is defined %}
   - keyserver: {{ repo.key_server }}
   {%- endif %}
+  {%- if repo.key_url is defined and repo.key_url.startswith('salt://') %}
```
Comment (epcim) [inline linux/system/repo.sls]: we should support even http:// and https:// anyway the condition to startswith is not much logical, on the other hand we should get rid of the curl usage above

Comment (bbinet) [reply linux/system/repo.sls]: http/https are supported already with the curl command I think

Comment (epcim) [reply linux/system/repo.sls]: Will be fixed properly with this: https://github.com/salt-formulas/salt-formula-linux/pull/155

**Pull request closed without merging**